facia/app/services/dotcomrendering/FaciaPicker.scala

```diff
@@ -51,6 +51,39 @@ object FrontChecks {
       "news/most-popular",
     )
 
+  val UNSUPPORTED_THRASHERS: Set[String] =
+    Set(
+      "https: //content.guardianapis.com/atom/interactive/interactives/thrashers/2022/12/wordiply/default",
+      "https: //content.guardianapis.com/atom/interactive/interactives/thrashers/2022/04/australian-election/default",
+      "https: //content.guardianapis.com/atom/interactive/interactives/thrashers/2021/07/full-story/default",
+      "https: //content.guardianapis.com/atom/interactive/interactives/thrashers/2021/10/saved-for-later/default",
+      "https: //content.guardianapis.com/atom/interactive/interactives/thrashers/2022/12/documentaries-signup-thrasher/default",
+      "https: //content.guardianapis.com/atom/interactive/interactives/thrashers/2021/12/100-best-footballers/default",
+      "https: //content.guardianapis.com/atom/interactive/interactives/thrashers/2021/01/football-weekly-thrasher/thrasher",
+      "https: //content.guardianapis.com/atom/interactive/interactives/2022/11/20/football-interactive-atom/knockout-full",
+      "https: //content.guardianapis.com/atom/interactive/interactives/thrashers/2021/07/pegasus/default",
+      "https: //content.guardianapis.com/atom/interactive/interactives/thrashers/2022/07/lakeside/default",
+      "https: //content.guardianapis.com/atom/interactive/interactives/thrashers/2022/07/support-guardian-thrasher/default",
+      "https: //content.guardianapis.com/atom/interactive/interactives/thrashers/2022/02/pw-uk/default",
+      "https: //content.guardianapis.com/atom/interactive/interactives/thrashers/2022/11/comfort-eating-grace-dent-thrasher-no-logo/default",
+      "https: //content.guardianapis.com/atom/interactive/interactives/thrashers/2022/02/weekend-podcast-2022/default",
+      // We can support those once we support full width for thrashers: https://github.com/guardian/dotcom-rendering/issues/7678
```

Comment:
Does this comment just apply to the thrasher immediately below on line 71? If not, I wonder if it's worth tagging each of the full-width ones separately (or putting an explicit end to the list of full-width thrashers?) because otherwise it's unclear where the list ends and other non-full-width ones might get added to the end later

Reply:
Thanks for the comment! Edited this - hopefully now it's clearer that it applies to all the thrashers below but let me know if not! Tried to leave space between 69 and 70 but `scalafmt` was not happy!